samples/bluetooth/central_uart/prj.conf

```diff
@@ -36,3 +37,12 @@ CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_FLASH_MAP=y
 CONFIG_NVS=y
 CONFIG_SETTINGS=y
+
+# Config logger
+CONFIG_LOG=y
+CONFIG_LOG_MINIMAL=n
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_UART=n
+CONFIG_LOG_PRINTK=n
+CONFIG_LOG_DEFAULT_LEVEL=3
```

Comment:
These are default values, you don't need these entries:
```
CONFIG_LOG_MINIMAL=n
CONFIG_LOG_PRINTK=n
CONFIG_LOG_DEFAULT_LEVEL=3
```

Reply:
Only `CONFIG_LOG_PRINTK=n` is no needed